### DISCUSSION TITLE
could anyone tell me how to override ctrl alt del?

### DISCUSSION THREAD

**Discussion by Skye310** [CosmosOS/Cosmos#1672] [Q&A]
I want to override ctrl alt del with my own function. How do i do this?

**Comment by quajak** [comment]
Its fixed in the devkit. While using the userkit its not possible to fix the problem since its directly in the Keyboard code.

**Reply by Skye310** [reply]
i am using userkit

**Reply by quajak** [reply]
Then its impossible, so you have to wait for new userkit or update to devkit.
